src/systemctl/systemctl.c

```diff
@@ -211,6 +211,9 @@ static int acquire_bus(BusFocus focus, sd_bus **ret) {
         if (getenv_bool("SYSTEMCTL_FORCE_BUS") > 0)
                 focus = BUS_FULL;
 
+        if (getenv_bool("SYSTEMCTL_FORCE_LOCAL") > 0)
+                focus = BUS_TRANSPORT_LOCAL;
```

Comment:
`focus` and `arg_transport` are two different things. You are assigning a value defined for the transports to the focus here, which really isn't right.

I figure you mean to say "focus = BUS_MANAGER" here. That said, BUS_MANAGER is what we should be using by default for most operations anyway, and for that we use the direct connection anyway already, no?